infra/test/infra_test.go

```diff
@@ -132,13 +132,23 @@ func EnableDestroyService(t *testing.T, terraformOptions *terraform.Options) {
 		},
 		WorkingDir: "../../",
 	})
+	shell.RunCommand(t, shell.Command{
+		Command: "sed",
+		Args: []string{
+			"-i.bak",
+			"s/deletion_protection = var.is_temporary ? \"INACTIVE\" : \"ACTIVE\"/deletion_protection = \"INACTIVE\"/g",
+			"infra/modules/identity-provider/main.tf",
+		},
+		WorkingDir: "../../",
+	})
```

Comment:
I don't think you need this in `infra_test.go`. `infra_test.go` runs things in a separate workspace which automatically marks the environment as temporary. I think you want to run this in `template_infra_test.go` which runs things in the main workspace since it simulates an actual project setup process, and it seems you already did that by updating destroy-app-service in template-only-bin.

Reply:
This failed for me. Let me see if I can dig up why.

Reply:
I think this is leftover from before #649. Tested my hypothesis on this theory with #668. Tested removing it in https://github.com/navapbc/platform-test/pull/108 and works fine.